### ISSUE TITLE
Transaction builder does not fetch implementation contract ABI for transparent proxy

### ISSUE THREAD

**Issue by CindyLo0** [5afe/safe-support#162]
### Issue Category

Other

### What happened?

- Using transaction builder to build a transaction onto an EIP-1967 Transparent Proxy does not include the implementation contract ABI
- See screenshot, 
![image](https://user-images.githubusercontent.com/79137197/226929242-d9b23f7e-97e4-41ce-87c4-378fef8ac1bc.png)
- https://etherscan.io/address/0x0000000000a39bb272e79075ade125fd351887ac#writeProxyContract

### Date and time this happened or you first noticed this issue

22-03-2023

### Network

Ethereum Mainnet

### Token Type

None

**Comment by mmv08** [comment]
The contract in question is Blur NFT marketplace which doesn’t work well with smart contract wallets, so I guess no safe interacted with it. However, because the safe transaction service only includes ABIs of contracts that safes previously interacted with, it needs to have one for Blur.

Also, the contract isn’t verified in our fallback source Sourcify.
https://sourcify.dev/server/files/1/0x17584A148d27aC5D06d87771464dAcbAF625Ce45

**Comment by mmv08** [comment]
Tl;dr: blur team should verify the contract on sourcify
